packages/env/src/index.ts

```diff
@@ -10,7 +10,7 @@ const environment = process.env.NODE_ENV || 'development'
 const __dirname = fileURLToPath(import.meta.url)
 const FILE_PUBLIC_PATH = 'uploads'
 
-if (environment !== 'production') {
+if (environment === 'development' || environment === 'test') {
```

Comment:
in staging env vars are defined in the docker compose, in production they come from the environment ofc